spec/03-plugins/20-hmac-auth/03-access_spec.lua

```diff
@@ -108,6 +108,23 @@ describe("Plugin: hmac-auth (access)", function()
       assert.equal(SIGNATURE_NOT_VALID, body.message)
     end)
 
+    it("should not be authorized when the HMAC signature is not properly base64 encoded", function()
+      local date = os.date("!%a, %d %b %Y %H:%M:%S GMT")
+      local hmacAuth = [["hmac username="bob",algorithm="hmac-sha1",]]
+        ..[[headers="date",signature="not really a base64 encoded value!!!"]]
```

Comment:
Looks much better, I just ran a few debug statements and realized what the test was doing 😅